host/bcm283x/gpio.go

```diff
@@ -326,12 +329,17 @@ func (p *Pin) FastOut(l gpio.Level) {
 // PWM pins
 //
 // PWM0 is exposed on pins 12, 18 and 40.
+// However, PWM0 is used for generating clock for DMA and unavailable for PWM.
+// TODO(simokawa): Use other DREQ source like PCM.
```

Comment:
Is it relevant to user documentation?

Reply:
Good point, I will update the document tomorrow.